tutorials/README.md

```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
 
 Once you are happy with staging, rerun the [Deploy
 tutorials](https://github.com/Qiskit/documentation/actions/workflows/deploy-tutorials.yml)
-workflow, but this time choose "Learning platform (production)". Warning: this will
-update every non-network tutorial to use the version from the `main` branch. That means
-that if another author had a tutorial that was merged to `main` but not yet ready to go live
-to production, you might accidentally deploy their tutorial. So, before deploying to
-production, check with the team that it is okay to deploy.
+workflow, but this time choose "Learning platform (production)". Warning: this
+will overwrite every non-network tutorial with the version from the `main`
+branch.
 
 After deploying to production, check https://learning.quantum.ibm.com/catalog/tutorials
 to ensure your tutorial is working correctly.
 
-## Gotcha: tutorial headings ignored
+## Gotcha: Tutorial titles
 
-One potential gotcha is that the learning platform ignores the top-level
-heading of the notebook. These headings are only included in the notebook for
-writers' convenience. If you want to change the title of a notebook, find the
-page on https://learning-api.quantum.ibm.com/admin and change its title
-there. Make sure to update the title in the notebook too.
+The learning platform ignores the top-level heading of the notebook. These
+headings are only included in the notebook for writers' convenience.
+
+> [!CAUTION]
+> The title dictates the URL slug. Do not change a tutorial's title without
+> having appropriate redirects set up.
+
+If you want to change the title of a notebook, find the page in the CMS and
+change its title there. Make sure to update the title in the notebook too.
 
 ## Adding new tutorials
 
 Each tutorial has its own folder in `tutorials/`. Within that folder is the content notebook
 (ending in `.ipynb`) and an optional `images` folder.
 
-To add a new tutorial to the learning platform, go to
-https://learning-api.quantum.ibm.com/admin/ and choose "Create item" (the blue
-`+` in the top-right corner). Enter all the information, but leave the content
-field blank. This will create the tutorial in "draft" mode.
+Once you've decided to add a new tutorial to the learning platform, a person
```

Comment:
could you make it clear in this intro that this process should first be done in staging before in prod